src/plays/cricket-game/styles.css

```diff
@@ -0,0 +1,289 @@
+@import url('https://fonts.googleapis.com/css2?family=Raleway:wght@100;200;300;400;500;600;700;800;900&family=Signika+Negative:wght@300;400;500;600;700&display=swap');
+
+/* enter styles here */
+
+* {
```

Comment:
No * override is allowed. Use the main class name `.cricket-home-body`

Reply:
Fixed!
https://github.com/reactplay/react-play/pull/972/commits/c7dba9eedf67f7176c2f523e63b23b6d6720673e